testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/security/callerprincipal/GetCallerPrincipalTestCase.java

```diff
@@ -97,11 +95,6 @@
 
     public static final String LOCAL_USER = "$local";
 
-    @BeforeClass
-    public static void beforeClass() {
-        AssumeTestGroupUtil.assumeElytronProfileTestsEnabled();
```

Comment:
@honza889 There's still an intermittent failure in GetCallerPrincipalTestCase that @mjurc and I are looking into so this one will still need to be ignored for now when running with the Elytron profile enabled.